src/styles/default-theme.css

```diff
@@ -211,6 +211,10 @@
 
   /*region Color Decisions(border)*/
 
+  /* Usage: for contrasted borders.
```

Comment:
nitpick: Could we try to order the variables? Could be by thickness first. Or could be intent first.